front-end/src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+// Styles
+import styles from './Timer.module.scss';
+
+const Timer = ({ running }) => {
+  const [time, setTime] = useState(0);
+
+  useEffect(() => {
+    let interval;
+    if (running) {
```

Comment:
Rewrite to `if (running) {
      interval = setInterval(() => {
        setTime((prevTime) => prevTime + 10);
      }, 10);
return;
    } 
      clearInterval(interval);
    `

Reply:
Done